python/cac_tripplanner/destinations/views.py

```diff
@@ -63,3 +64,35 @@ def get(self, request, *args, **kwargs):
 
         matched_objects = Destination.objects.filter(point__within=iso, published=True)
         return HttpResponse('{{"matched": "{0}"}}'.format(matched_objects), 'application/json')
+
+class SearchDestinations(View):
+    """ View for searching destinations via an http endpoint """
+
+
+    def get(self, request, *args, **kwargs):
+        """ GET destinations that match search queries
+
+        Must pass either:
+          - lat + lon params
+          - text param
+
+        A search via text will return destinations that match the destination name
+        A search via lat/lon will return destinations that are closest to the search point
+
+        """
+        params = request.GET
+        lat = params.get('lat', None)
+        lon = params.get('lon', None)
+        text = params.get('text', None)
+        output_fields = ('name', 'description', 'point', 'address', 'city', 'state', 'zip')
+
+        results = []
+        if lat is not None and lon is not None:
```

Comment:
If `lat` and `lon` are strings here, might as well just do:
```python
if lat and lon:
```
(will be truthy if non-null and non-empty).